reference/checkouts.v3.yml

```diff
@@ -265,13 +265,12 @@ paths:
                     quantity: 1
         - name: include
           in: query
-          required: false
           x-enum-elements:
             - name: Enum_consignments.availableShippingOptions
               description: ''
           type: string
           description: Must be included to get available shipping options
-          default: consignments.available_shipping_options
+          default: available_shipping_options
```

Comment:
I think `consignments.available_shipping_options` is correct and doesn't need to change.
Basically, if `?include=consignments.availableShippingOptions` is added to the request query param the response will contain list of available shipping options. Otherwise, response will not have it.

Reply:
Thank you Sepehr, I am glad you saw this. I will communicate you comments to the requestor.